### PR TITLE
[MPLUGIN-434] Improve dependency management

### PR DESCRIPTION
**Pull request by slawekjaranowski**
- excludes in one place
- exclude plexus-container-default
- exclude plexus-component-api

### CODE REVIEW

File: maven-plugin-report-plugin/pom.xml [hunk 1]
```diff
@@ -240,6 +237,7 @@
                                 <maven.compiler.source>${maven.compiler.source}</maven.compiler.source>
                                 <maven.compiler.target>${maven.compiler.target}</maven.compiler.target>
                             </properties>
+                            <junitPackageName>maven.invoker.it.plugin.report</junitPackageName>
```
Comment (gnodet) [inline maven-plugin-report-plugin/pom.xml:240]: That looks unrelated. What is is used for ?

**Review by gnodet — COMMENTED** [review]
LGTM apart from the `junitPackageName` which looks unrelated

File: maven-plugin-report-plugin/pom.xml (hunk 1 above)
Comment (slawekjaranowski) [reply maven-plugin-report-plugin/pom.xml:240]: Ok, not related.

It is used for package name of IT tests like 
https://ci-maven.apache.org/job/Maven/job/maven-box/job/maven-plugin-tools/job/master/lastCompletedBuild/testReport/maven.invoker.it/

Comment (slawekjaranowski) [reply maven-plugin-report-plugin/pom.xml]: removed

**Comment by jira-importer** [comment]
Resolve #743
